nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-service-api/src/main/java/org/apache/nifi/serialization/JsonRecordReaderFactory.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.serialization;
+
+public interface JsonRecordReaderFactory extends RecordReaderFactory {
+
+}
```

Comment:
It's related to the comment left by @turcsanyip https://github.com/apache/nifi/pull/6350#pullrequestreview-1091713871

In it's current implementation we need a `JsonTreeReader` CS to read the schema of the records. Since Airtable returns a JSON response it made sense to limit the available choices for the "Schema Reader" to only `JsonTreeReader` and `PropertyDescriptor.identifiesControllerService` seems to work only with interfaces. That's why I created a new interface.

If we move away from using a `JsonTreeReader` to parse the schema this interface can be removed

Reply:
@dam4rus This interface can now be deleted because the Recored Reader approach has been reverted.